lib/orderbook/OrderBookRepository.ts

```diff
@@ -17,15 +17,25 @@ class OrderbookRepository {
     return this.models.Pair.findAll({ raw: true });
   }
 
-  async getOrders(): Promise<Orders> {
+  async getOrders(maxResults?: number, pairId?: string): Promise<Orders> {
+    const whereClauseBuy: any = { quantity: { [Op.gt]: 0 }, peerId: { [Op.ne]: null } };
+    const whereClauseSell: any = { quantity: { [Op.lt]: 0 }, peerId: { [Op.ne]: null } };
+
+    if (pairId !== undefined && pairId !== '') {
```

Comment:
i think if (pairId) will be sufficient here. in case empty string are getting here, we need to stop them beforehand.